combustion-android-ble/src/main/java/inc/combustion/framework/service/DeviceManager.kt

```diff
@@ -497,6 +497,13 @@ class DeviceManager(
      */
     fun stopDfuMode() = service.stopDfuMode()
 
+    /**
+     * State flow containing the firmware details for all nodes on the network.
+     */
+    fun getNetworkFirmwareState(): StateFlow<FirmwareState>? {
```

Comment:
@nick-sasquatch -- this adds the flow for firmware information for network to be used on the device list screen.

Reply:
Thanks for getting this in there!